src/libsync/theme.cpp

```diff
@@ -674,8 +674,9 @@ QString Theme::versionSwitchOutput() const
 bool Theme::isDarkColor(const QColor &color)
 {
     // account for different sensitivity of the human eye to certain colors
-    double treshold = 1.0 - (0.299 * color.red() + 0.587 * color.green() + 0.114 * color.blue()) / 255.0;
-    return treshold > 0.5;
+    // https://stackoverflow.com/questions/3942878/how-to-decide-font-color-in-white-or-black-depending-on-background-color
```

Comment:
Hm... why that change, this looks fairly equivalent, isn't it?

Reply:
The treshold is a little bit lower (to get white icons on nextcloud blue) and I thought it would be easier to read if there is no divide and subtraction. But I can revert it?

Reply:
How much lower in practice? I admit I'm always having a hard time wrapping my head around threshold which aren't in the [0.0;1.0] interval so that's why I reacted mostly. :-)

When you say a bit lower for the threshold, it's quite a bit or am I confused? Moved from 0.5 to 0.28 (1.0 - 185/256) isn't it?

Reply:
Now that it's clear that we don't need support for dark/light theme switch in the wizard, I wonder if we should just hardcode the icons that we need and leave that function untouched. But maybe then we will have problems if a branded customer chooses a different background color...

Reply:
Yes the threshold is quite a bit lower.

Reply:
Yes, this would cause issues in branding if we just drop this.

Reply:
I'm still on the fence regarding dropping this or not... The [0;255] vs [0.0;1.0] is a matter of personal taste in the end (and my preference is obvious) so that doesn't matter that much. The change of threshold on the other hand I wonder if that wouldn't cause unexpected side effects because now for existing branded clients some will switch from white to black for the icons once that PR gets in... while they might have chosen a very light text already today.

Reply:
Yes, it's a bad idea to change it. I have already encountered problems.

Reply:
I'm a little bit out of ideas how to set it then wo white on Nextcloud blue. We could do something really ugly like checking it its the Nextcloud blue color and then set it to white, otherwise let isDarkColor decide which icon to pick.

Reply:
Ah so the reason you changed initially was that it'd pick black icons on the Nextcloud blue? Suddenly I understand why you changed it in the first place. :-)

Reply:
I guess if it's "just" about needing to force white for the Nextcloud blue... it's a bit meh but just special case it. We can't really change the threshold for all branded clients like this.

Reply:
@er-vin @FlexW 

Maybe we need to use this "threshold-based" method as a fall-back, but, apply the customer's selected color if it is set in Brander?

Reply:
@allexzander I like the idea, but I I think this would involve some code changes. We then need to introduce some enum or similar which says if dark icons, light icons or threshold based icons should be used?